### ISSUE TITLE
RangeDisplay/RangeDisplay.toc:30 Duplicate File Load Detected.

### ISSUE THREAD

**Issue by JH0Ni-github** [mitchnull/RangeDisplay#4]
```
RangeDisplay/RangeDisplay.toc:30 Duplicate File Load Detected. [RangeDisplay/libs/LibDataBroker-1.1-4/LibDataBroker-1.1.lua] first loaded at [RangeDisplay/RangeDisplay.toc:28]
```

**Comment by JH0Ni-github** [comment]
Another addon fixed this by just removing the duplicate line (line 30).

**Comment by mitchnull** [comment]
Should be fixed in v6.0.7
